### DISCUSSION TITLE
Tab Scrollable

### DISCUSSION THREAD

**Discussion by ja-vi-app** [radix-ui/primitives#1811] [Help] [answered]
Hello

I am discovering more about Radix Primitives and faced a problem related to responsiveness.

I have a tab component and can have a lot of topics, which makes some cases overflow. The company's designer suggested making tabs scrollable. I tried some ways, but none succeeded.

I want something like the website Radix Primitives has, like a carousel, but with tabs.

I think it isn't necessary for this question, but I am currently using `styled-components` with `Next JS`.

![image](https://user-images.githubusercontent.com/111320603/205336796-3fe3b7e4-f373-476f-a89d-2f2ac29d9682.png)

**Comment by joaom00** [comment, marked as answer]
Do you want scroll with mouse/touch?

**Reply by ja-vi-app** [reply]
Yes if possible

**Reply by joaom00** [reply]
You can see the component [here](https://github.com/radix-ui/website/blob/main/components/marketing/Carousel.tsx)

**Reply by ja-vi-app** [reply]
Thank you very much, sorry for delayed answer
